pkg/serverless/trace/trace.go

```diff
@@ -37,6 +38,9 @@ func (l *LoadConfig) Load() (*config.AgentConfig, error) {
 // Start starts the agent
 func (s *ServerlessTraceAgent) Start(enabled bool, loadConfig Load) {
 	if enabled {
+		//this will fail the grpc client and allow to skip hostname resolution
```

Comment:
I would expand this comment slightly to explain a bit more about why we are doing this